src/definitions/collections/grid.less

```diff
@@ -1105,12 +1105,12 @@
 .ui.relaxed[class*="vertically divided"].grid > .row:before {
   margin-left: (@relaxedGutterWidth / 2);
   margin-right: (@relaxedGutterWidth / 2);
-  width: ~"calc(100% - "@relaxedGutterWidth~")";
+  width: calc(~'100% - @relaxedGutterWidth');
```

Comment:
This will output the variable in css, since `~" "` wraps a string literal

outputted css
```css
width: calc(100% - @relaxedGutterWidth);
```